vertical-pod-autoscaler/hack/vpa-process-yamls.sh

```diff
@@ -39,7 +39,7 @@ if [ $# -gt 2 ]; then
 fi
 
 ACTION=$1
-COMPONENTS="vpa-crd vpa-beta-crd vpa-rbac updater-deployment recommender-deployment admission-controller-deployment"
+COMPONENTS="vpa-beta-crd vpa-rbac updater-deployment recommender-deployment admission-controller-deployment"
```

Comment:
I think we need to decide whether VPA beta should take care about switching off VPA alpha.

If no - vpa-crd.yaml can be removed after all (unless we want to keep it for manual clean-up - but then this needs to be documented somewhere).
If yes - we need to add vpa-crd to COMPONENTS if ACTION==delete.

Reply:
I plan on adding a switch-me-to-beta script or instruction. Added vpa-crd on delete, thanks.